lib/link.js

```diff
@@ -11,8 +11,16 @@ export default class Link extends Component {
 
   static propTypes = {
     children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.element
+      PropTypes.element,
+      (props, propName) => {
```

Comment:
The reason I Added a custom prototype check is that in #937 I'm going to look into stripping prototype validation on production builds 👍🏻. In the issue it's one of the requirements that it Doesn't log in production / the code shouldn't. be there :)